### DISCUSSION TITLE
Support of the cheap HTU21D sensor

### DISCUSSION THREAD

**Discussion by speedypidz** [ESPresense/ESPresense#1099] [Ideas]
Hi Darrell,

Thank you so much for your great work, and sorry for my english. I have a lot of cheap HTU21D that i use with esphome. My question is that is there a way to use it with espresence and if not is it possible for you to had it in the project ?

Thank you

Nicolas

**Comment by DTTerastar** [comment]
Almost all the sensors have been added by contributions.  I am mainly focused on improving the core mission of accurate positioning.

**Reply by speedypidz** [reply]
Thank you Darrel for your answer, I will have a look to the code a see if I can do something with my pour skill 😅
